conda_smithy/templates/main.yml.tmpl

```diff
@@ -0,0 +1,18 @@
+on:
+  status: {}
+  check_run:
+    types:
+      - completed
```

Comment:
We may want check_suite here instead of check_run. I am still investigating. Let's hold off on this PR until that happens.

Reply:
Ok. check_run should be changed to check_suite here.